docs/admin-guide/zope-manager-users.md

```diff
@@ -107,3 +107,45 @@ When you run the script, if the user already exists:
     Maybe the user already exists and nothing is done then.
     Or the implementation does not give info when it succeeds.
     ```
+### `addzopeuser` script in dockerized setup
+
+#### Prerequisites
+-   A running docker swarm stack from the full featured docker based Plone Volto deployment 
+-   Created by current Cookieplone project template including backend, frontend, postgres, traefik, varnish:
+
+#### Create a new Zope Manager User with a non existing user-ID
```

Comment:
"with a non existing user-ID" is a bit confusing here. This addzopeuser command is the same as the one mentioned above, it's just a different procedure for running it inside the Docker container. so we can copy some of the same explanations from above about how it behaves if the user already exists.